.github/workflows/release.yml

```diff
@@ -96,13 +96,14 @@ jobs:
           BIN_NAME=hdx-${{ matrix.code-target }}
           mv target/${{ matrix.target }}/release/hdx $BIN_NAME
 
+      - run: chmod +x hdx-*
```

Comment:
I needed `zed::make_file_executable` in the extension, which we don't need to do; if we `chmod +x` the binary. 

I believe these file modes are transfered through the assets; but wasnt too sure so kept the zed fn api to `+x` it.